cmd/exporters/influxdb/influxdb.go

```diff
@@ -51,66 +51,70 @@ func (e *InfluxDB) Init() error {
 	}
 
 	var (
-		url, addr, port, bucket, org, v, p string
-		err                                error
+		url, addr, bucket, org, token, v, p *string
+		port                                *int
 	)
 
 	// check required / optional parameters
 
-	if bucket = e.Params.GetChildContentS("bucket"); bucket == "" {
+	if bucket = e.Params.Bucket; bucket == nil {
 		return errors.New(errors.MISSING_PARAM, "bucket")
 	}
-	e.Logger.Debug().Msgf("using bucket [%s]", bucket)
+	e.Logger.Debug().Msgf("using bucket [%s]", *bucket)
 
-	if org = e.Params.GetChildContentS("org"); org == "" {
+	if org = e.Params.Org; org == nil {
 		return errors.New(errors.MISSING_PARAM, "org")
 	}
-	e.Logger.Debug().Msgf("using organization [%s]", org)
+	e.Logger.Debug().Msgf("using organization [%s]", *org)
 
-	if e.token = e.Params.GetChildContentS("token"); e.token == "" {
+	if token = e.Params.Token; token == nil {
 		return errors.New(errors.MISSING_PARAM, "token")
 	} else {
```

Comment:
else block not needed since if does a return

Reply:
actually in else block it's assigning token to e.token, which would be used later

Reply:
What I meant was the return in the `if` means you're code can be made more clear like this:
```
if token = e.Params.Token; token == nil {
    return errors.New(errors.MISSING_PARAM, "token")
}
e.token = *token
e.Logger.Debug().Msg("will use authorization with api token")
```

Reply:
ohh, I understood different way. Got it.